### DISCUSSION TITLE
Contributing to crowdin's localization, can I use already done enlgish localization as a base?

### DISCUSSION THREAD

**Discussion by lucassmcardoso** [misskey-dev/misskey#11172] [Q&A]
Giving some context Initially, I'm new to to the FOSS community and wish to contribute to the misskey project starting small with localization. I'm from Brazil and do not know much japanese, so I was thinking of using the already accepted english transalation that is available on crowdin. Is there any problem or can I do it? If the response is positive I'll open an issue to ask for the addition of a  PT-BR localization on crowdin.

**Comment by mappi-pr** [comment]
@lucassmcardoso 
I think you can deal with it in the same way as this issue.
https://github.com/misskey-dev/misskey/issues/11331

**Comment by tassoman** [comment]
When translations are missing, it should fallback automatically to English. If also this isn't available, then falls back to Japanese. (rarely!)

If you mean better context of translatable object, then I suggest to copy paste the Japanese string into Google Translate and get literal translation. Sometimes English version it's too much simplified, if your language have more descriptive or male/female words, you'd better leverage on this.  
In Italian, I usually can be descriptive as for Japanese original meaning, using same length of English words.

Always remember to use the much more inclusive words, for women and men also. Enjoy!
